app/controllers/api/v1/user/users_controller.rb

```diff
@@ -6,4 +6,22 @@ def show
     render json: current_user
   end
 
+  def finance_overview
```

Comment:
What are your thoughts on scope `user:read` vs. `finance:user`?
Since this really is a financial overview, I would slightly favour the latter, with `user:read` more for user details and membership. What do you think?

Reply:
i have no clear thought about our scopes yet. i'll go with your proposal

Reply:
:+1:
In that case I would propose an ordergroup controller in the API that includes this method.